CacheMultimap.adoc

```diff
@@ -0,0 +1,79 @@
+= Native cache Mutimap
+
+The goal is to provide Infinispan Native CacheMultimap support. The proposal here is to start with the smallest interface implementation, in order to provide a proper API to the first multimap client, vert-x project.
+However, this API will evolve, and this first implementation must work on Embedded and Server mode.
+This first Multimap won't support duplicate values on the same key. 
+
+[source, java]
+.CacheMultimap.java
+----
+public interface CacheMultimap<K, V> {
+   boolean put(K key, V value); // <1> <2>
+
+   Collection<V> get(K key); // <3>
```

Comment:
The original email thread mentioned that "this version" doesn't support repeated entities but I believe they meant that this is a temporary limitation?

If it's a temporary milestone only, it should be `Collection` so to allow getting rid of the limitation soon.

If it's not a temporary limitation and indeed this should return a `Set` .. maybe this whole API shouldn't be called `MultiMap` ? That would be very confusing...